src/throttler/throttler.ts

```diff
@@ -75,6 +74,16 @@ export class Queue<T> {
     }
   }
 
+  /**
+   * @return {boolean} true if there are unscheduled task waiting to be scheduled.
```

Comment:
I missed this: remove the type - Typescript already knows the type from the signature.

```suggestion
   * @return `true` if there are unscheduled task waiting to be scheduled.
```